api/wallet/spec/definitions/DepositRevertStatus.yaml

```
@@ -1,14 +1,15 @@
+---
 type: object
 properties:
   status:
     description: |
-      Статус отмены поступления денежных средств.
+      Deposit revert status.
 
-      | Значение    | Пояснение                                               |
+      | Meaning     | Explanation                                             |
       | ----------- | ------------------------------------------------------- |
-      | `Pending`   | Отмена в процессе выполнения                            |
-      | `Succeeded` | Отмена поступления средств произведёна успешно          |
-      | `Failed`    | Отмена поступления средств завершилась неудачей         |
+      | `Pending`   | Deposit revert in progress                              |
+      | `Succeeded` | Deposit revert of receipt of funds was successful       |
+      | `Failed`    | Deposit revert of incoming funds failed                 |
```

Если следовать DepositAdjustmentStatus, стоит сформулировать так для идентичности
```suggestion
      | `Succeeded` | Deposit revert completed successfully       |
      | `Failed`    | Deposit revert failed                 |
```